operations/filter_kgraph_discrete_attribute.yml

```diff
@@ -0,0 +1,41 @@
+id: filter_kgraph_discrete_attribute
```

Comment:
Rename this "filter_kedge_discrete_attribute" or broaden the definition to cover knodes, too.

Reply:
changed this to `filter_kgraph_discrete_kedge_attribute`. Does that work?